mavlink:
  # how often should the driver send a heartbeat to the gimbal
  heartbeat_period: 1.0 # seconds
  # how often should the driver send a command to the gimbal (recommended is 20Hz)
  command_period: 0.05 # seconds

  resend:
    # after no ACK is received for this duration, try to resend
    duration: 0.1
    max_tries: 10

  # this is the default system and component ID for an autopilot
  driver:
    system_id: 1
    component_id: 240

  # this is the default system and component ID for gimbals
  gimbal:
    system_id: 1
    component_id: 0

# Example useful streams that can be requested from SimpleBGC gimbals include:
# enum name                         enum value    returned stream
# MAV_DATA_STREAM_RAW_SENSORS       1             RAW_IMU
# MAV_DATA_STREAM_RC_CHANNELS       3             RC_CHANNELS_SCALED
# MAV_DATA_STREAM_RAW_CONTROLLER    4             ATTITUDE
# MAV_DATA_STREAM_EXTRA1            10            ATTITUDE
# MAV_DATA_STREAM_EXTRA2            11            ATTITUDE
stream_requests:
  ids: [10]
  rates: [200]

mount:
  stabilize_roll: true
  stabilize_pitch: true
  stabilize_yaw: true
  # Possible input modes:
  # enum name             enum value
  # angle_body_frame      0
  # angular_rate          1
  # angle_absolute_frame  2
  roll_input_mode: 0
  pitch_input_mode: 0
  yaw_input_mode: 0
